src/testModel.ts

```diff
@@ -83,14 +83,18 @@ export class TestModel {
   constructor(vscode: vscodeTypes.VSCode, workspaceFolder: string, configFile: string, playwrightInfo: { cli: string, version: number }, options: TestModelOptions) {
     this._vscode = vscode;
     this._options = options;
-    this.config = { ...playwrightInfo, workspaceFolder, configFile };
     this._useLegacyCLIDriver = playwrightInfo.version < 1.44;
     this._playwrightTest =  this._useLegacyCLIDriver ? new PlaywrightTestCLI(vscode, this, options) : new PlaywrightTestServer(vscode, this, options);
     this._didUpdate = new vscode.EventEmitter();
     this.onUpdated = this._didUpdate.event;
+    this.config = { ...playwrightInfo, workspaceFolder, configFile };
```

Comment:
Why did this move?

Reply:
I was initially tring to add `serverUrlPrefix` into the config and for that I needed the `PlaywrightTestServer`. But then I realized that I could only get the value asynchronously and aborted that approach, but I forgot to undo the move. I'll rollback it